Gemfile

```diff
@@ -73,4 +73,6 @@ group :production do
   gem "pg"
 end
 
-gem 'dotenv'
\ No newline at end of file
+gem 'dotenv'
+
+gem 'rubocop', '~> 1.60', require: false
```

Comment:
and also generally it's a good practice to put those gems in the development group, you don't need them everywhere because you use them only in development env